spec/feature_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'takeaway'
+
+describe 'User Stories' do 
+# as a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+  it 'so I can check if I want to order something, I would like to see a list of dishes'do
+  takeaway = Takeaway.new
+  menu = Menu.new
+  expect(takeaway.list_menu).to include ("Roast Pork Belly Hirata Buns")
+  end
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+
+
+  it 'so I can the meal I want, I would like select some number of dishes available'do
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)